app/spell_check.py

```diff
@@ -9,6 +10,17 @@ def parse_txt(raw_input):
             word_list.append(word)
     return word_list
 
+def parse_txt_other_lang(raw_input):
+    init_word_list = re.split(' ', raw_input)
+    word_list = []
+    for word in init_word_list:
+        if word and word != ' ':
+            if word[-1] in '\.\?!,;_:%“”\' …\+"’‘':
+                word_list.append(word[0:len(word)-1])
+                word_list.append(word[-1])
```

Comment:
Do we want these non-alphanumeric characters in the word list?

Reply:
These are characters that the user has entered on the frontend to spell check, we will need them in the wordlist to print them back to the front

Reply:
Oh I see my confusion. word_list is currently containing the valid words and all the other white space and symbols.

In order to align with the other logic that we have in place I think that we will need to have this parser return 2 array. One with all input text (spaces, symbols, etc) and the other that contains all the indices in that input that are considered valid words. Take a look at the original parsing function. If the two parsers return different formats, it will make things more challenging to integrate. Thoughts?

Reply:
Yeah, this will need reworked. I had based it and its returns off of the parse text function present in the branch which only returns the word list and nothing else. I'll get this reworked